Hardcore.lua

```diff
@@ -643,24 +628,21 @@ function Hardcore:Debug(msg)
 end
 
 -- Alert UI
-function Hardcore:ShowAlertFrame(style, message)
-	-- style is a key-based property of ALERT_STYLES
+function Hardcore:ShowAlertFrame(styleConfig, message)
 	-- message is any text accepted by FontString:SetText(message)
 
 	message = message or ""
 
-	local frame, text, icon, file, filename, delay = nil, nil, nil, nil, nil, nil
-	local data = ALERT_STYLES[style] or ALERT_STYLES["hc_red"]
```

Comment:
This line was always defaulting to the default `ALERT_STYLES["hc_red"]`

Reply:
If it works, great - thanks for the fix